src/main/java/com/openmrs/migrator/Migrator.java

```diff
@@ -0,0 +1,193 @@
+    }
+
+    if (run) {
+      executeRunCommandLogic();
+    }
+    return Optional.empty();
+  }
+
+  private void runAllJobs() throws IOException {
+    try {
+      for (String t : jobs) {
+
+        InputStream xml = fileIOUtilities.getResourceAsStream(t);
+        pdiService.runJob(xml);
+      }
+    } catch (KettleException e) {
+      // Do nothing kettle prints stack trace
+    }
+  }
+
+  private void executeSetupCommand() throws IOException {
+    List<String> pdiFiles = new ArrayList<>();
+
+    pdiFiles.add("pdiresources/jobs/merge-patient-job.kjb");
+    pdiFiles.add("pdiresources/transformations/merge-patient.ktr");
+    pdiFiles.add("settings.properties");
+
+    bootstrapService.createDirectoryStructure(dirList);
+    bootstrapService.populateDefaultResources(pdiFiles);
+  }
+
+  private void executeRunCommandLogic() throws FileNotFoundException, IOException {
+    int choice = ConsoleUtils.startMigrationAproach(console);
+
+    switch (choice) {
+      case 1:
+        Optional<String> providedDataBaseName = ConsoleUtils.getDatabaseDetaName(console);
+        Optional<String> storedDataBaseName =
+            fileIOUtilities.searchForDataBaseNameInSettingsFile(providedDataBaseName.get());
+
+        if (!storedDataBaseName.isPresent()) {
+          if (ConsoleUtils.isConnectionIsToBeStored(console)) {
+            fileIOUtilities.addSettingToConfigFile(providedDataBaseName.get());
+          }
+          fileIOUtilities.setConnectionToKettleFile(providedDataBaseName.get());
+        }
+        runAllJobs();
+        break;
+
+      case 2:
+        String selectDBName =
+            ConsoleUtils.getValidSelectedDataBase(
+                console,
+                validateDataBaseNames(fileIOUtilities.getValueFromConfig("password", "=")));
+        if (selectDBName != null) {
+          fileIOUtilities.setConnectionToKettleFile(selectDBName);
+          runAllJobs();
+        }
+
+        break;
+      case 3:
+        String dbName =
+            ConsoleUtils.getValidSelectedDataBase(
+                console,
+                new HashSet<>(
+                    dataBaseService.getDatabases(
+                        fileIOUtilities.getValueFromConfig("password", "="))));
+        fileIOUtilities.setConnectionToKettleFile(dbName);
+        runAllJobs();
+        break;
+
+      case 4:
+        List<Path> inputs = fileIOUtilities.listFiles(Paths.get("input/"));
+
+        String sqlDumpFile = ConsoleUtils.chooseDumpFile(console, inputs);
+
+        if (sqlDumpFile == null) {
+          break;
+        }
+
+        String databaseName = ConsoleUtils.getChosenDBName(console);
+
+        dataBaseService.importDatabaseFile(databaseName, sqlDumpFile);
+        fileIOUtilities.setConnectionToKettleFile(databaseName);
+        runAllJobs();
+
+        break;
+
+      default:
+        ConsoleUtils.showUnavailableOption(console);
+        break;
+    }
+  }
+
+  private Set<String> validateDataBaseNames(String password)
```

Comment:
lets place this in the DatabaseService (with unit tests)

Reply:
done